tools/constants.csv

```diff
@@ -12,6 +14,9 @@ false,double,earth_perihelion_time,-6287982.0
 false,double,earth_period,365.256363004 * 24.0 * 60.0 * 60.0
 false,lin::Vector4d,q_eci_perifocal,0.127456632677880,-0.158124280715206,0.762378784011859,-0.614434787689722
 false,double,b_noise_floor,0.0
-false,double,max_mtr_moment,0.113337 / 2.0
-false,lin::Matrix3x3f,JB_single_sat,0.0333f,0.0f,0.0f,0.0f,0.0333f,0.0f,0.0f,0.0f,0.0067f
-false,lin::Matrix3x3f,JB_docked_sats,0.0333f,0.0f,0.0f,0.0f,0.0333f,0.0f,0.0f,0.0f,0.0067f
\ No newline at end of file
+false,float,max_mtr_moment,0.113337f / 2.0f
+true,lin::Matrix3x3f,J_sat,0.0333f,0.0f,0.0f,0.0f,0.0333f,0.0f,0.0f,0.0f,0.0067f
```

Comment:
I just wanted to make note of this here really quick, I think that FSW should be concerned about changing up our moment of inertia externally. Seems to make more sense to me at least (same with the gains).

Reply:
Do you mean that FSW should take care of creating a system for changing the values if required? Sure, if you could open a ticket we could track that work?

Reply:
Yep sounds good - was going to open one up!

Reply:
I'm not sure about using global variables to control the pointing controller behavior. Shouldn't the moment of inertia be owned by the estimator as part of its internal state, then set or get with functions? Also when running a sim with two sats on the same computer, will the global variable be shared by both leader and follower?